tests/test_core.py

```diff
@@ -599,7 +599,7 @@ def test_get_multile_column_primary_key(engine_testaccount):
         assert columns_in_mytable[1]["primary_key"], "primary key"
 
         primary_keys = inspector.get_pk_constraint("mytable")
-        assert primary_keys["constrained_columns"] == ["gid", "id"]
+        assert primary_keys["constrained_columns"] == ["id", "gid"]
```

Comment:
note: this change is because we updated the implementation to stay complied with the sqlalchemy spec to keep column ordering.
I treat this as a bug fix instead of breaking change